docs/unofficial-mac-broker-integration.md

```diff
@@ -0,0 +1,19 @@
+# An unofficial doc of mac broker integration
+
+
+### Steps
+1. Install CP (company portal) on the mac, login your acounts in CP and make sure MDM succeeds.
+2. Install msal: `pip install msal==1.27.0b2`
+3. Update pymsalruntime to the newest: `pip install pymsalruntime`
```

Comment:
How would this install be handled in practice? Would both be installed as a dependency for Azure CLI?

Reply:
> How would this install be handled in practice? Would both be installed as a dependency for Azure CLI?

Effectively yes. [Azure CLI makes explicit dependency on `msal[broker]`](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/requirements.py3.Darwin.txt#L106) which in turn pulls in a proper `PyMsalRuntime` as dependency, too.

@fengga , please change your PR's base branch to `1.27.0b2`, rather than `dev`. Besides, I doubt people (us included) would remember to read this new "unofficial doc". If you intend to make sure we always test with latest `PyMsalRuntime`, a better tactic may be to either (1) use a test script which creates a new virtual environment every time, so that it will pull in the latest `PyMsalRuntime`; or (2) write a unit test case ([similar to this one](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/dev/tests/test_cryptography.py#L36-L42)) to error out when old PyMsalRuntime is detected.

Reply:
@rayluo , there is no branch or tag with name `1.27.0b2`. Is it time to enable mac broker for dev branch?

Reply:
> I found there is no branch or tag with name `1.27.0b2`.

See the fine-print in the [release notes for `1.27.0b2`](https://github.com/AzureAD/microsoft-authentication-library-for-python/releases/tag/1.27.0b1). :-P

> Is it time to enable mac broker for dev branch?

The original plan was to wait for at least one major downstream partner to test and adopt this feature first. I remember that @oldalton engaged Azure CLI team for this recently. We do have an outside customer who successfully tested the `1.27.0b2` beta version, though.

Reply:
Given broker adoption is now an SFI metric, w3 can consider that plan to be changed.

Reply:
Per offline conversation with @rayluo , Ray will merge https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/596 first, then let's revisit this doc PR.